rakelib/education_benefits_submission.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+desc 'update all records in education_benefits_submissions table'
+task education_benefits_submission: :environment do
+  # `update_all` is being used because the `vettec` field will reset to `false`
+  # as the form isn't live on staging and prod
+  # rubocop:disable Rails/SkipsModelValidations
+  EducationBenefitsSubmission.update_all(vettec: false)
```

Comment:
> This method constructs a single SQL UPDATE statement and sends it straight to the database.
[source](https://apidock.com/rails/v4.0.2/ActiveRecord/Relation/update_all)
Didn't know this. Nice.